packages/@sfx/search-plugin/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as SearchPlugin } from './search-plugin';
```

Comment:
Also export the new interfaces. Make sure to add the appropriate tests.

```suggestion
export { default as SearchPlugin, SearchPluginExposedValue, SearchPluginOptions } from './search-plugin';
```